platform/ui/src/components/userPreferencesForm/HotKeysPreferences.js

```diff
@@ -10,31 +10,14 @@ import PropTypes from 'prop-types';
 
 export class HotKeysPreferences extends Component {
   static propTypes = {
-    hotKeysData: PropTypes.objectOf(
-      PropTypes.shape({
-        keys: PropTypes.arrayOf(PropTypes.string).isRequired,
-        label: PropTypes.string.isRequired,
-      })
-    ).isRequired,
-    onChange: PropTypes.func,
+    hotkeyDefinitions: PropTypes.array.isRequired,
```

Comment:
Might be nice to be more explicit about this. `PropTypes.arrayOf(PropTypes.shape({}))`